### ISSUE TITLE
.find() return only first value from array

### ISSUE THREAD

**Issue by ygruninBGGK** [balderdashy/sails#2907]
i have the array in document. 

While using the:
 UserBusinessConnection.find({user: req.session.me}).populate("business").exec(function findOneCB(err,businenesses){
			  console.log('We found '+businenesses);

It brings only first value from array "business" .
If i user .native - it works very well - but the problem there to use .populate

Any help?

**Comment by sailsbot** [comment]
Thanks for posting, @ygruninBGGK.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
